SchoolHousing/SchoolHousing/Pages/Index.cshtml

```diff
@@ -6,34 +6,63 @@
     var schoolData = (ElementarySchoolResponse.ElementarySchool[])ViewData["ElementarySchool"];
 }
 
+    <html>
+    <head>
+        <style>
+            body {
```

Comment:
Nice styling, but move to a linked stylesheet to declutter this page and make the styles usable elsewhere.

Also note, the head section already exists for this page, in the template that includes all of the Razor pages.  Adding a head section here is going to create duplicate head sections, which is not desirable, does not comply with XHTML 1.0 standard, and may lead to unpredictable results.   Did you test this out first?